posts/2021-11-30-RedHat-Connect-London.adoc

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "A day out in London at the RedHat Connect Summit"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/yasmin-aumeeruddy
+author_github: https://github.com/yasmin-aumeeruddy
+seo-title: A day out in London at the RedHat Connect Summit
+seo-description: 
+blog_description: 
+open-graph-image: https://openliberty.io/img/AndyMcCright-1.png
+additional_authors:
+- name: Grace Jansen
+  github: https://github.com/GraceJansen
+  image: https://avatars3.githubusercontent.com/GraceJansen
+---
+= A day out in London at the RedHat Connect Summit
+Yasmin Aumeeruddy <https://github.com/yasmin-aumeeruddy>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+link:https://twitter.com/gracejansen27[Grace Jansen] and link:https://github.com/yasmin-aumeeruddy[Yasmin Aumeeruddy] from the Open Liberty team joined our cousins at RedHat to participate in the link:https://www.redhat.com/en/summit/connect/emea/london/developers[RedHat Connect Summit in London]. This event was a single-day event, offering a variety of developer-focused sessions, many of which utilised or focused on OSS projects and initiatives. The topics of the sessions ranged from those focused on cloud-native development, to DevOps, to enterprise development, to more general language and framework sessions. As well as the conference sessions there were also lightning talks occurring in the expo area and live coding and arcade games as well! These one-day events are run in cities all over the globe. You can check out if there's an event occurring near you at their link:https://www.redhat.com/en/summit/connect#locations--dates[website].
```

Comment:
suggested edit:
This event was a single-day event, offering a variety of developer-focused sessions, many of which utilised or focused on OSS projects and initiatives. The topics of the sessions ranged from those focused on cloud-native development, to DevOps, to enterprise development, to more general language and framework sessions.

--->

This single-day event offered a variety of developer-focused sessions, many of which focused on open source software (OSS) projects and initiatives. The topics included cloud-native development, DevOps and  enterprise development, as well as more general language and framework sessions.


As well as the conference sessions there were also lightning talks occurring in the expo area and live coding and arcade games as well!
--->
The summit also featured lightning talks in the expo area, live coding, and even arcade games!